pandas/tests/indexes/test_multi.py

```diff
@@ -632,6 +632,13 @@ def test_from_arrays_index_series_period(self):
 
         tm.assert_index_equal(result, result2)
 
+    def test_from_arrays_different_lengths(self):
+        # GH13599
+        idx1 = [1, 2, 3]
```

Comment:
@jreback is that test really required? The same input array length case is anyway getting checked in all other `from_arrays` related tests.